xhr/send-redirect.htm

```diff
@@ -27,10 +33,10 @@
           client.send(null)
         })
       }
-      redirect("301")
-      redirect("302")
-      redirect("303")
-      redirect("307")
+
+      for (var number = 300; number <= 399; number++) {
```

Comment:
Let's not make this 100 tests, but pick a few numbers to illustrate the problem.

Reply:
I changed to 300-310, 350, 399. Let's hope that no special handling for any of the other codes occurs.

Reply:
If you find any we should add it, but this seems good enough for now.